### PR TITLE
Remove RemovedOn attribute from ECOINFO cluster

### PR DESCRIPTION
**Pull request by tehampson**
There is conflicting spec behavior for what is supposed to happen when a device is removed. Spec issue filed  https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/10234

This is a provisional cluster, as such it is allowed to change in ways that break compatibility

### CODE REVIEW

File: examples/fabric-bridge-app/fabric-bridge-common/src/BridgedDeviceManager.cpp
Comment (tehampson) [inline examples/fabric-bridge-app/fabric-bridge-common/src/BridgedDeviceManager.cpp:116]: Blocking comment on https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/10234 being resolved first

File: examples/fabric-bridge-app/fabric-bridge-common/src/BridgedDeviceManager.cpp
Comment (tehampson) [inline examples/fabric-bridge-app/fabric-bridge-common/src/BridgedDeviceManager.cpp:117]: Blocked on https://github.com/CHIP-Specifications/chip-test-plans/pull/4501, merging first

**Review by andy31415 — APPROVED** [review]
Approving, assuming spec PR goes through.

**Review by woody-apple — APPROVED** [review]
Same

**Review by woody-apple — APPROVED** [review]
Same

**Comment by tehampson** [comment]
>Approving, assuming spec PR goes through.

It is now merged in https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10298

**Pull request merged**
